x-pack/platform/plugins/shared/streams/server/lib/streams/client.ts

```diff
@@ -136,7 +136,8 @@ export class StreamsClient {
     await this.upsertStream({
       request: {
         dashboards: [],
-        stream: omit(rootStreamDefinition, 'name'),
+        description: rootStreamDefinition.description,
+        stream: omit(rootStreamDefinition, ['name', 'description']),
```

Comment:
this is probably not what we want. `name` is omitted because of how ES works (you don't define an index in the body either when you're updating it)